aspnetcore/host-and-deploy/windows-service/sample/AspNetCoreService.csproj

```diff
@@ -1,24 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net461</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.1.1" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.0.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="2.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.1.0" />
```

Comment:
As I was going through this example I noticed I wasn't able to access some of the new 2.1 features like HttpClientFactory on IServiceCollection. I had to change this line of code to include Microsoft.AspNetCore.App to make it happen - which I believe is required if you're migrating from 2.0 to 2.1 (according to the migration doc)

Reply:
@Eldorian It's not strictly speaking required (in this case, you could probably just add `Microsoft.Extensions.Http`), but it does simplify package management and is what we normally pitch for `netcoreapp2.1` apps. Prior versions of this sample listed packages individually, but I don't see a good reason to stick to that approach.

@Tratcher Any objections on moving to ...

```xml
<PackageReference Include="Microsoft.AspNetCore.App" />
<PackageReference Include="Microsoft.AspNetCore.Hosting.WindowsServices" Version="2.1.0" />
```

Reply:
Ah, thanks for that explanation on what I could have done to make it work - I'm pretty new to ASP.NET Core in general and got into it shortly before 2.1 released, so everything I've been reading in the documents to migrate from 2.0 to 2.1 mention Microsoft.AspNetCore.App that in the package reference. 

Just makes more sense to me then to have it be consistent as I'm going through different documentation :)

Reply:
The basics on .NET Standard and package management are over in the .NET Core docs. That doc set kicks off over at: https://docs.microsoft.com/dotnet/core/